test/dynamo/test_repros.py

```diff
@@ -3270,6 +3270,29 @@ def f():
 
         self.assertEqual(f(), _GLOBAL_CPU_TENSOR + _GLOBAL_CPU_TENSOR)
 
+    @torch._inductor.config.patch("layout_optimization", True)
```

Comment:
@wconstab got a repro! It just took a bit more work than I thought in order to get inductor to actually change the output stride of the convolution. I needed:
- layout_optimization = True
- channels dim to be > 64 (from the inductor code [here](https://github.com/pytorch/pytorch/blob/main/torch/_inductor/graph.py#L325))

Reply:
great, thanks @bdhirsh!!